Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -436,6 +436,7 @@ JS::Value Document::run_javascript(const StringView& source)
         parser.print_errors();
         return JS::js_undefined();
     }
+    document().interpreter().clear_exception();
     return document().interpreter().run(document().interpreter().global_object(), *program);
```

Comment:
Wouldn't it be nicer to do an exception check *after* running code, and then clearing it?

Reply:
yeah that makes sense :)